FastSimulation/TrackingRecHitProducer/python/TrackingRecHitProducer_cfi.py

```diff
@@ -1,58 +1,33 @@
-        MergingProbabilityForwardFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fmergeprob.root'),
-        MergedPixelBarrelResolutionXFile = cms.string('FastSimulation/TrackingRecHitProducer/data/bxsmear.root'),
-        MergedPixelForwardResolutionXFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fxsmear.root'),
-        MergedPixelBarrelResolutionYFile = cms.string('FastSimulation/TrackingRecHitProducer/data/bysmear.root'),
-        MergedPixelForwardResolutionYFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fysmear.root'),
-        templateId = cms.int32( 40 ),
-        select=cms.string("subdetId==BPX"),
-    )
-)
+# Phase 1 Era: replace plugins by Phase 1 plugins
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+trackingPhase1.toModify(fastTrackerRecHits, plugins = pixelPluginsPhase1)
 
-fastTrackerRecHits.plugins.append(
-    cms.PSet(
-        name = cms.string("pixelForwardSmearer"),
-        type=cms.string("PixelForwardTemplateSmearerPlugin"),
-        BigPixelBarrelResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/NewPixelResolutionBarrel38T.root'),
-        EdgePixelBarrelResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/NewPixelResolutionBarrelEdge38T.root'),
-        RegularPixelBarrelResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/PixelBarrelResolution2014.root'),
-        BigPixelForwardResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/NewPixelResolutionForward38T.root'),
-        EdgePixelForwardResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/NewPixelResolutionForward38T.root'),
-        RegularPixelForwardResolutionFile = cms.string('FastSimulation/TrackingRecHitProducer/data/PixelForwardResolution2014.root'),
-        UseCMSSWPixelParametrization = cms.bool(False),
-        MergeHitsOn = cms.bool(False),
-        MergingProbabilityBarrelFile = cms.string('FastSimulation/TrackingRecHitProducer/data/bmergeprob.root'),
-        MergingProbabilityForwardFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fmergeprob.root'),
-        MergedPixelBarrelResolutionXFile = cms.string('FastSimulation/TrackingRecHitProducer/data/bxsmear.root'),
-        MergedPixelForwardResolutionXFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fxsmear.root'),
-        MergedPixelBarrelResolutionYFile = cms.string('FastSimulation/TrackingRecHitProducer/data/bysmear.root'),
-        MergedPixelForwardResolutionYFile = cms.string('FastSimulation/TrackingRecHitProducer/data/fysmear.root'),
-        templateId = cms.int32( 41 ),
-        select=cms.string("subdetId==FPX"),
-    )
-)
+# Phase 2 Era: replace plugins by Phase 2 plugins, etc...
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingPhase2PU140.toModify(fastTrackerRecHits, plugins = pixelPluginsPhase2)
```

Comment:
Same here, please use `phase2_tracker` modifier instead.

Reply:
@makortel : Done!

Reply:
@pmaksim1 you need to fix the imports as well ('Configuration.Eras.Modifier_trackingPhase2PU140_cff' -> 'Configuration.Eras.Modifier_phase2_tracker_cff')